stdlib/messaging/rabbitmq/src/main/java/org/ballerinalang/messaging/rabbitmq/RabbitMQUtils.java

```diff
@@ -59,6 +61,20 @@ private static MapValue populateRabbitMQErrorRecord(String msg) {
                 RabbitMQConstants.RABBITMQ_ERROR_DETAILS, valueMap);
     }
 
+    public static ArrayList<ObjectValue> addToList(ArrayList<ObjectValue> arrayList, ObjectValue objectValue) {
```

Comment:
Shall we add java doc comments to these public methods

Reply:
Fixed